plugins/repository-s3/build.gradle

```diff
@@ -302,8 +320,13 @@ task s3Fixture(type: AntFixture) {
   env 'S3FIXTURE_TEMPORARY_KEY', "${s3TemporaryAccessKey}"
   env 'S3FIXTURE_TEMPORARY_SESSION_TOKEN', "${s3TemporarySessionToken}"
   env 'S3FIXTURE_EC2_BUCKET_NAME', "${s3EC2Bucket}"
+  env 'S3FIXTURE_EC2_CREDENTIALS', "${s3EC2Credentials}"
```

Comment:
There's no need for a Gstring here, could have used the  `s3EC2Credentials` variable directly.

Reply:
good spot - thanks @atorok 👍